### DISCUSSION TITLE
Sessions die unexpectedly

### DISCUSSION THREAD

**Discussion by SteveGiralt** [scandum/tintin#97] [Bugs]
Hello,

I am having an issue where my session dies without warning. 

- connect to my mud
- leave my computer connected for some amount of time (an hour plus typically)
- When I return the screen has not changed, upon pressing any key the session "catches up" and shows it died previously

This is not due to in game idling. In the past with:
- Same OS, hardware, and setup
- Same mud
- Same configuration

I could leave my session connected indefinitely and all was good.

I'm wondering if there is a way to get a debugging log to help troubleshoot this issue, or perhaps I should roll back to an older version?

I completely uninstalled and reinstalled TT++ - currently on the latest version, built per the directions on the official website. Running on Ubuntu 18.04.6 LTS

Screenshot attached showing I read the poster and then went idle. When I returned I pressed enter and you can see what happened.

Thanks

![Screenshot_20220427-134440.png](https://user-images.githubusercontent.com/88471484/165616025-c378b4f5-fdd3-4ca9-bcbf-02eaae09d045.png)

**Comment by scandum** [comment]
This could be due to a change at the ISP level, but let me know if going back to an older version fixes the problem.

If it's due to a change at the ISP level the common workaround is to send something to the MUD once every 10 minutes.

**Reply by SteveGiralt** [reply]
I hadn't considered ISP issues. I'll try setting a ticker and sending commands every 10 minutes or so. 

Thanks for your response and the awesome client. I'll update here with what I find.

**Comment by SteveGiralt** [comment]
Just following up - I have a ticker running now and all seems to be going well. Thanks again!
